digitalidentity/share_session_notification_builder.go

```diff
@@ -0,0 +1,62 @@
+package digitalidentity
+
+import (
+	"encoding/json"
+)
+
+// ShareSessionNotification specifies a ShareSessionNotification for ShareSession
+type ShareSessionNotification struct {
+	url       string
+	method    string
+	verifyTls bool
```

Comment:
i think this might have to be `*bool` so we don't default this to `false` (without them explicitly asking for it) and then later on check if nil and set a default (i think it's meant to be `true` by default?) 

(As a side note, this API maybe should have been `skipTLSVerification` or something so the default is false.)